### PR TITLE
[BACKLOG-16402] Doc reviews

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon changes related with the spike to implement doc reviews

### CODE REVIEW

File: impl/client/src/main/doc-js/pentaho/IRange.jsdoc [hunk 1]
```diff
@@ -15,21 +15,20 @@
  */
 
 /**
- * A range represents an interval on an underlying, base, ordinal domain.
+ * A range represents an interval on an underlying, ordinal domain.
```
Comment (nantunes) [inline impl/client/src/main/doc-js/pentaho/IRange.jsdoc]: That comma doesn't make sense if the apposition is gone.

File: impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc [hunk 2]
```diff
@@ -15,14 +15,14 @@
  */
 
 /**
- * The `IContext` interface exposes contextual information of the Pentaho Web Client Platform,
- * like, for example, its
+ * The `IContext` interface allows access to the Pentaho Web Client Platform.
```
Comment (davidmsantos90) [inline impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc]: @pentaho/millenniumfalcon I'm not sure with the change from exposes to allows access, what is your opinion?

Comment (dcleao) [inline impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc]: "Allows access" would be ok, if "to contextual information" hadn't been removed — that changes the meaning.

Comment (davidmsantos90) [inline impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc]: Thanks for the feedback :)
this was an attempt of mine, to make this uniform with doc suggestion in the other two files, I will change this after feedback from the doc team to a suggestion I gave taking into account what you said

**Pull request merged**